src/panels/developer-tools/statistics/dialog-statistics-adjust-sum.ts

```diff
@@ -42,7 +42,7 @@ export class DialogStatisticsFixUnsupportedUnitMetadata extends LitElement {
         lastMoment ||
         `${now.getFullYear()}-${
           now.getMonth() + 1
-        }-${now.getDate()} ${now.getHours()}:${now.getMinutes()}:${now.getSeconds()}`,
+        }-${now.getDate()} ${now.getHours()}:00:00`,
       amount: 0,
```

Comment:
I want to change this to be the new value. And we then calculate under the hood the adjustment. As a user you see the spike and don't want to calculate "spike - usual" ah yes that's what I want to enter.